tests/test_parser.py

```diff
@@ -1,11 +1,13 @@
 import os
 import sys
+from typing import Generic
```

Comment:
I don't think this is needed?

Reply:
Probably. VS Code likes importing random stuff from auto-completes